docs/index.md

```diff
@@ -3,28 +3,34 @@ myst:
   html_meta:
     "description": "The Volto user interface (UI) is a React-based frontend for the Plone CMS. It is the default user interface starting with the release of Plone 6."
```

Comment:
Let's combine into a single sentence, removing version number, "default", and "CMS".

```suggestion
    "description": "The Seven user interface (UI) is a React-based frontend for Plone."
```